client-app/src/desktop/common/SampleTreeGridModel.js

```diff
@@ -55,15 +59,26 @@ export class SampleTreeGridModel {
         });
     }
 
-    async doLoadAsync(loadSpec) {
+    async doLoadAsync({isRefresh, isAutoRefresh}) {
         const {gridModel, dimChooserModel} = this,
             dims = dimChooserModel.value;
 
         return XH.portfolioService
             .getPositionsAsync(dims, true)
             .then(data => {
-                gridModel.loadData(data);
-                gridModel.selectFirst();
+                if (isRefresh || isAutoRefresh) {
```

Comment:
just FYI -- should be enough to just check if isRefresh is true.  `isAutoRefresh` implies `isRefresh`